README.md

```diff
@@ -1 +1,57 @@
-# vba-migrator-addin
+# Project Title
+### vba-migrator-addin
+VBA Editor add-in to show compatibility with Google Sheets.
+
+## About
+For now this Add-in will give you a button in your VBA editor.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+```
+Having Window OS
```

Comment:
Windows. How about Excel itself? You can say the version of windows you have as you have not tested on on other versions.

Reply:
Also mention the version of excel

Reply:
Done.